tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -52,7 +60,7 @@ IncludeByKey.PLATFORM {
 
 # include DMIC config if needed.
 IncludeByKey.NUM_DMICS {
-	"[1-4]"	"platform/intel/dmic-generic.conf"
+	"[1-4]"	"platform/intel/dmic-generic-add-dmic16k.conf"
```

Comment:
Maybe we can split DMIC0 and DMIC1 into 2 conf files, dmic0-generic.conf for dmic0 and dmic16k-generic.conf for dmic16k. 
DMIC1 may be used by other purpose, such as WoV. If we split DMIC0/1 into different conf files. It will be easy to add such support. We just make some judgement in the top file, such as cavs-nocodec.conf to pick up the right configuration, like:
```
# If there is DMICs, DMIC0 is enabled by default
IncludeByKey.NUM_DMICS {
	"[1-4]"	"platform/intel/dmic-generic.conf"
}
# enable dmic16k if DMIC16 is defined
IncludeByKey.DMIC16k {
	"1"	"platform/intel/dmic16k.conf"
}
# enable dummy_wov if DUMMY_WOV is defined
# DUMMY_WOV and DMIC16 are mutually exclusive
IncludeByKey.DUMMY_WOV {
	"1"	"platform/intel/dmic-wov.conf"
}
```

Reply:
Yep. I hope I can work in Q1 next year with tplg2. I would add topology and pipelines those enable testing of all playback and capture processing.